cmd/restic/cmd_prune.go

```diff
@@ -232,24 +241,19 @@ func printPruneStats(printer progress.Printer, stats repository.PruneStats) erro
 	if stats.Size.Unref > 0 {
 		printer.V("unreferenced:                    %s\n", ui.FormatBytes(stats.Size.Unref))
 	}
-	totalBlobs := stats.Blobs.Used + stats.Blobs.Unused + stats.Blobs.Duplicate
-	totalSize := stats.Size.Used + stats.Size.Duplicate + stats.Size.Unused + stats.Size.Unref
-	unusedSize := stats.Size.Duplicate + stats.Size.Unused
```

Comment:
Why does unused also include duplicate? Should this be saved in `stats.Size.Unused` such that this is also used in the JSON output?

Reply:
Reading #5212 this seems intended.

Keeping unused and duplicate separate values in the stats (which are also used for JSON output).